exo-run/src/app-runner.ls

```diff
@@ -1,7 +1,7 @@
 require! {
   'chalk': {red}
   'events' : {EventEmitter}
-  '../../exosphere-shared' : {DockerHelper}
+  '../../exosphere-shared' : {DockerCompose}
```

Comment:
These changes are tied to this PR. This PR is really just implementing Dockerode, but DockerHelper had to be renamed to DockerCompose to distinguish it from the Dockerode processes which uses that name. There has been no extra logic added to DockerCompose.